centreon/src/CentreonRemote/Infrastructure/Service/PollerInteractionService.php

```
@@ -144,6 +144,32 @@ private function moveConfigurationFiles(array $pollerIDs): void
                 }
+
+                // Send VMWARERESTART only if VMWare config has changed
+                $vmwareStatement = $this->db->prepare(
+                    'SELECT vmware_updated FROM nagios_server WHERE id = :pollerId'
+                );
+                $vmwareStatement->bindValue(':pollerId', (int) $host['id'], \PDO::PARAM_INT);
+                $vmwareStatement->execute();
+                $vmwareRow = $vmwareStatement->fetch(\PDO::FETCH_ASSOC);
+                if ($vmwareRow && $vmwareRow['vmware_updated'] === '1') {
+                    if (isset($host['localhost']) && $host['localhost'] == 1) {
+                        exec(escapeshellcmd('sudo -n -- systemctl restart centreon_vmware'), $restartOutput, $restartReturnCode);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$restartOutput'. (undefined)

(UnusedLocalVariable)